be/src/common/config.h

```diff
@@ -987,7 +987,8 @@ CONF_mInt64(lake_pk_compaction_min_input_segments, "5");
 // Used for control memory usage of update state cache and compaction state cache
 CONF_mInt32(lake_pk_preload_memory_limit_percent, "30");
 CONF_mInt32(lake_pk_index_sst_min_compaction_versions, "2");
-CONF_mInt32(lake_pk_index_sst_max_compaction_bytes, /*1GB*/ "1073741824");
+CONF_mInt32(lake_pk_index_sst_max_compaction_versions, "100");
+CONF_mInt32(lake_pk_index_base_cumulative_compaction_ratio, "10");
```

Comment:
add comments to this configure item. which compares which to get this ratio.

Also I may prefer use cumulative_size / base_size so it could be, if the ratio >= 0.1, always do base compaction, and below 0.1 always cumulative, unless the base_size can be 0?

Reply:
Yeah, I pick `base / cumulative` instead of `cumulative / base` at first considering the risk of dividing 0

Reply:
I can change it to `cumulative / base`.

Reply:
Done